include/models.h

```diff
@@ -98,6 +102,8 @@ class Model {
 
     bool setStopWords(std::vector<std::vector<int>> stopWordsList);
 
+    bool freeSeqs(std::vector<int> seqIDs);
```

Comment:
why not consider passing the parameter by reference?

Reply:
done.